src/main/java/ca/corefacility/bioinformatics/irida/ria/web/ajax/dto/ajax/AjaxUpdateItemSuccessResponse.java

```diff
@@ -0,0 +1,17 @@
+package ca.corefacility.bioinformatics.irida.ria.web.ajax.dto.ajax;
+
+public class AjaxUpdateItemSuccessResponse extends AjaxResponse {
```

Comment:
JavaDoc

Reply:
Updated in a7b64f193de58ec912b52536d2aaa704a43a136f